exercises/E-conditionals/exercise.js

```diff
@@ -8,6 +8,16 @@
 
 let name = "Daniel";
 let danielsRole = "mentor";
+    
+function roleCheck (danielsRole) {
+if (danielsRole === "mentor") {
```

Comment:
Try to avoid when function parameters have the same name as global variables - it confuses the reader a lot and may result in very subtle bugs in your code. I'd recommend to pick a more generic parameter name:

```javascript
function roleCheck (role) {
  // ...
}
```

Second comment - function names _usually_ start with a verb (printMessage, getItem, updateDatabase, checkRole). There are some exceptions where you can break this rule, but I would stick with it by default. I'd suggest to rename your function to `checkRole`